src/CodeGenerator.php

```diff
@@ -91,6 +93,7 @@ public static function generateFromDescriptor(
      * @param ?string $grpcServiceConfigJson Optional grpc-serv-config json string.
      * @param ?string $gapicYaml Optional gapic configuration yaml string.
      * @param ?string $serviceYaml Optional service configuration yaml string.
+     * @param bool $numericEnums Optional whether to generate the numeric-enums JSON encoding system parameter.
```

Comment:
nit: consider expanding this a little for clarity: "whether to include in requests the system parameter asking for JSON-encoded responses to encode enum values as numbers"

Reply:
Done.